apis/fluent-en_GB/extensions/jdk8/atrium-api-fluent-en_GB-jdk8-jvm/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/jdk8/PathAssertionsSpec.kt

```diff
@@ -9,6 +9,8 @@ import java.nio.file.Path
 class PathAssertionsSpec : ch.tutteli.atrium.specs.integration.PathAssertionsSpec(
     fun0(Expect<Path>::exists),
     fun0(Expect<Path>::existsNot),
+    fun1(Expect<Path>::startsWith),
+    fun1(Expect<Path>::startsNotWith),
     fun1(Expect<Path>::endsWith)
 ){
     @Suppress("unused", "UNUSED_VALUE")
```

Comment:
please add startsWith/startsNotWith also to ambiguityTest (and please rebase your branch on master)

Reply:
Requested tests added and also rebased.